verification/testing/verification.go

```diff
@@ -166,6 +176,7 @@ var IrreversibleTestCases = []TestCase{
 	DeriveContextLocalityTestCase,
 	DeriveContextPrivilegeEscalationTestCase,
 	DeriveContextMaxTCIsTestCase,
+	DeriveContextRecursiveTestCase,
```

Comment:
Should this also include DeriveContextRecursiveOnDerivedContextsTestCase?

Reply:
Done